src/policy_metadata.rs

```diff
@@ -631,4 +636,30 @@ mod tests {
 
         assert!(metadata.validate().is_err());
     }
+
+    #[test]
+    fn metadata_with_min_kubewarden_version() -> Result<(), ()> {
```

Comment:
I don't think this test provides much value, this is basically testing serde ability to serialize to json something... :thinking: